std/assembly/number.ts

```diff
@@ -331,23 +345,23 @@ export abstract class F64 {
   static readonly NaN: f64 = NaN;
 
   static isNaN(value: f64): bool {
-    return builtin_isNaN<f64>(value);
+    return isNaN<f64>(value);
```

Comment:
Looks like it will conflict if we ever align the resolver with JS to also take the current function's name into account.

Reply:
What if I declare it as `function internal_isNaN` without export and in the end of file export as:
```ts
export { internal_isNaN as isNaN, internal_isFinite as isFinite }
```
?